examples/accesscontextmanager/serviceperimeterresource.yaml

```
@@ -0,0 +1,44 @@
+---
+apiVersion: cloudplatform.gcp.upbound.io/v1beta1
+kind: Project
+metadata:
+  annotations:
+    meta.upbound.io/example-id: cloudplatform/v1beta1/project
+    upjet.upbound.io/manual-intervention: "Requires organization level IAM permissions"
+  labels:
+    testing.upbound.io/example-name: project
+  name: project
+spec:
+  forProvider:
+    projectId: project
+    name: project
+
+---
+apiVersion: cloudplatform.gcp.upbound.io/v1beta1
+kind: ProjectService
+metadata:
+  annotations:
+    meta.upbound.io/example-id: cloudplatform/v1beta1/projectservice
```

```suggestion
    meta.upbound.io/example-id: accesscontextmanager/v1beta1/serviceperimeterresource
```
Same here, could you please fix `example-id`?

---

Done